### ISSUE TITLE
AOR_Reports - Feature request: COUNT subgroups

### ISSUE THREAD

**Issue by jellydonuts** [salesagility/SuiteCRM#1096]
![additional subgroup](https://cloud.githubusercontent.com/assets/1541720/13787566/b3ac77b2-ea99-11e5-85bc-bf43277fa077.jpg)

Hi there, referring to the image attached, it would really help if there were an option to provide a count on a subgroup of items like in the image (red circles). 

From what I've tried, the group & count functionality on anything other than the "Main Group", seems to do more than group, it actually reduces all rows down into one row and gives a count instead of "grouping" and counting.

I guess what I'm trying to ask for is the same functionality of the "Main Group" (top left of report creator) within each item's "group" checkbox (each row's grouping).

Let me know if further explanation is required.

**Comment by samus-aran** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [AOR_Reports - Feature request: COUNT subgroups](https://trello.com/c/E7X42kdf/114-aor-reports-feature-request-count-subgroups)

**Comment by samus-aran** [comment]
We have now set up a new home for suggestions at Trello.  All github issues that were labeled 'suggestion' have been moved and will be closed.  Certain ones will be progressed within the new Suggestion Box and may be re-opened.

Announcement of moving Suggestions:
https://suitecrm.com/forum/suggestion-box/13691-moving-suggestions-from-github-to

New SuiteCRM Suggestion Box
https://trello.com/b/Ht7LbMqw/suitecrm-suggestion-box
